TechlunchApp/Controllers/IngredientsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using TechlunchApp.ViewModels;
+
+namespace TechlunchApp.Controllers
+{
+    public class IngredientsController : Controller
+    {
+        private string apiUrl;
+
+        public IngredientsController(IConfiguration configuration)
+        {
+            apiUrl = configuration.GetValue<string>("apiUrl");
```

Comment:
Please use DI to pass the API URL here. Also, use string constant instead of using string "apiURL"